Components/Inputs/inputs.css

```diff
@@ -0,0 +1,74 @@
+@import url("../../default.css");
+
+/* form and input styles */
+.form-container {
+  width: fit-content;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-group {
+  padding: 0rem 2rem 2rem;
+  display: flex;
+  flex-direction: column;
+  background-color: var(--light-color);
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+
+.form-group button {
+  width: 10rem;
+}
+
+input,
+textarea {
+  display: inline-block;
+  padding: 8px 6px;
+  font-size: 1rem;
+  color: var(--dark-color);
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #ddd;
+  border-radius: 0.2rem;
+}
+
+input:disabled {
+  background-color: #ddd;
+  opacity: 0.7;
+  border: 1px solid var(--dark-color);
+}
+
+input[type="checkbox"],
+input[type="radio"] {
+  display: inline;
+  cursor: pointer;
+  accent-color: var(--primary-color);
+  width: 1rem;
+  height: 1rem;
+}
+
+textarea {
+  min-height: 160px;
+
+}
+
+.input-label {
+  display: inline-block;
+  margin: 0.25rem 0rem;
+  padding: 0.25rem;
+}
+
+.input-error {
+  border: 1px solid var(--danger-color);
+  outline: 1px solid var(--danger-color);
+}
+
+.validation-error {
```

Comment:
For Validation error, some bottom margin can be given to look more good.

Reply:
Thank. I will add that